mini-bank/withdraw.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Withdraw</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <link rel="stylesheet" href="../helpers/css/components.css" />
+    <link rel="stylesheet" href="mini-bank.css" />
+  </head>
+  <body>
+    <div class="container-fixed">
+      <section class="header-component">
+        <div class="navigations">
+          <div class="nav-buttons">
+            <ul>
+                <a href="transactions.html"
+                ><li>Transactions</li></a
+              >
+              <a href="deposits.html"><li>Deposit</li></a>
+              <a href="withdraw.html" class="active"><li>Withdraw</li></a>
+            </ul>
+            </ul>
+          </div>
+
+          <a href="user-profile.html">
+            <div class="user-nav">
+              <div class="user-icon"><i class="fas fa-circle-user"></i></div>
+              <div id="user-name" class="user-name">Username</div>
+            </div>
+          </a>
+        </div>
+
+        <div class="balance-wrapper">
+          Bal: &nbsp
+          <div class="user-balance" id="user-balance">0.00</div>
+        </div>
+      </section>
+
+      <div class="page-desc-holder">
+        <h1 class="page-desc">Withdraw</h1>
+      </div>
+
+      <div class="header-component">
+        <form id="deposit-form">
+          <div class="form-group">
+            <label>Amount to withdraw</label>
+            <input type="number" class="form-control" id="amount-to-withdraw" />
+          </div>
+          <div class="form-group">
+            <label>Pin</label>
+            <input type="password" class="form-control" id="user-pin" />
+          </div>
+          <button id="withdraw-button" class="btn-success btn-lg">
+            Submit
+          </button>
+        </form>
+      </div>
+
+      <div class="create-account" id="create-new-account">
```

Comment:
Like in deposit and transactions page, this button shouldn't be here as the user already has an account. If they didn't, they won't be able to access this page.